packages/url/src/filter-url-for-display.js

```diff
@@ -53,3 +53,14 @@ export function filterURLForDisplay( url, maxLength = null ) {
 		truncatedFile
 	);
 }
+
+/**
+ * Removes the protocol from a URL.
+ *
+ * @param {string} url - The URL to remove the protocol from.
+ *
+ * @return {string} The URL without the protocol.
+ */
+export function removeProtocol( url ) {
+	return url.replace( /^(?:https?:)\/\/(?:www\.)?/, '' );
```

Comment:
Question: why this regex is slightly different from [the one used in core](https://github.com/WordPress/wordpress-develop/blob/4b91a597b7a91d8ce099314a744fca5309dd0134/src/js/_enqueues/vendor/tinymce/plugins/wplink/plugin.js#L19)?

Reply:
Hi @afercia, I took the reference of the regex from [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/url/src/filter-url-for-display.js#L17) I don't know the exact reason why this is diferent from the one used in core